src/layout/Tree/TreeContext.ts

```diff
@@ -0,0 +1,18 @@
+import { ReactNode, createContext, useContext } from 'react';
+
+export interface TreeContextProps {
+  /**
+   * A custom icon to be used for collapsed nodes.
+   */
+  collapsedIcon?: any;
+
+  /**
+   * A custom icon to be used for expanded nodes.
+   */
+  expandedIcon?: any;
```

Comment:
Can the icon types be more specific?

Reply:
Agreed. We need to audit the types all over anyways.